### ISSUE TITLE
Page Can't be displayed

### ISSUE THREAD

**Issue by IoTecgh** [unosquare/passcore#251]
**PassCore Server**
- OS: [Windows ]
- Provider: [Active Directory ]

**Describe the bug**
After doing everything in the steps to setup passcore, I can't get to see the page loading. I get the "This page can't be displayed" error and I'm using IIS 8.5 and other sites on it are working.

Please will be glad if you can help me with this.

Thanks

**Comment by geoperez** [comment]
Any status?

**Comment by Bakanych** [comment]
Facing same issue. Trying to run v.3.6.2 using readme. IIS 7 on windows datacenter 2016 x64. No success, same symptoms.

**Comment by geoperez** [comment]
Can you elaborate? A screenshot with Chrome Debugger maybe.

**Comment by Bakanych** [comment]
I don't think app ever started. What I did:
- downloaded the latest release and unpacked it into c:/passcore3;
- installed dotnet core hosting from dotnet-hosting-2.1.1-win.exe installer
- setup apppool using your readme.md
- created application with source = c:/passcore3 and bindings http *  = 8080

When I try to open it I get:
This page isn’t working 192.168.0.1 is currently unable to handle this request.
HTTP ERROR 500

I also tried just to start app using exe binary and it works from console with kestrel.
